buildspec/release/20buildrelease.yml

```diff
@@ -19,13 +20,17 @@ phases:
             - apt-get install -y libasound2-plugins
     build:
         commands:
+            - echo "TARGET_EXTENSION=${TARGET_EXTENSION}"
             # --unsafe-perm is needed because we run as root
             - npm ci --unsafe-perm
-            - cp ./README.quickstart.vscode.md ./README.md
-            - npm run package
+            - |
+                if [ $TARGET_EXTENSION = 'toolkit' ]; then
+                  cp ./README.quickstart.vscode.md ./README.md
+                fi
+            - npm run package -w packages/${TARGET_EXTENSION}
+            - cp packages/${TARGET_EXTENSION}/package.json ./package.json
 
 artifacts:
     files:
-        - aws-toolkit-vscode*
-        - packages/toolkit/package.json
-    discard-paths: true
+        - '*.vsix'
+        - package.json
```

Comment:
don't we want `discard-paths: true` ?

Reply:
It's harmless though, right? And slightly beneficial because it makes our intentions explicit?